src/content/content-script.js

```diff
@@ -68,12 +68,13 @@ const ContentScript = {
 
   syncOverwriteProperties() {
     const overwrittenProperties = new Set([
-      { originalMethod: null, parentObject: window.navigator.mediaDevices, methodName: "getSupportedConstraints", potentiallyShowContextBanner: true },
-      { originalMethod: null, parentObject: window.navigator.mediaDevices, methodName: "enumerateDevices", potentiallyShowContextBanner: false },
-      { originalMethod: null, parentObject: window.navigator.mediaDevices, methodName: "getUserMedia", potentiallyShowContextBanner: true },
-      { originalMethod: null, parentObject: window.navigator.mediaDevices, methodName: "getDisplayMedia", potentiallyShowContextBanner: true },
-      { originalMethod: null, parentObject: window.navigator, methodName: "mozGetUserMedia", potentiallyShowContextBanner: true },
-      { originalMethod: null, parentObject: window.navigator, methodName: "mozGetUserMediaDevices", potentiallyShowContextBanner: true },
+      { originalMethod: null, parentObject: window.MediaDevices.prototype, methodName: "getSupportedConstraints", potentiallyShowContextBanner: true },
+      { originalMethod: null, parentObject: window.MediaDevices.prototype, methodName: "enumerateDevices", potentiallyShowContextBanner: false },
+      { originalMethod: null, parentObject: window.MediaDevices.prototype, methodName: "getUserMedia", potentiallyShowContextBanner: true },
+      { originalMethod: null, parentObject: window.MediaDevices.prototype, methodName: "getDisplayMedia", potentiallyShowContextBanner: true },
+      { originalMethod: null, parentObject: window.Navigator.prototype, methodName: "mediaDevices", potentiallyShowContextBanner: true },
```

Comment:
Why is this line getting added?  I don't think it will do anything particularly useful: the `mediaDevices` is not function-valued, so replacing it with a proxy that only defines `construct` and `apply` hooks will be a no-op, no?